packages-common/nanoflow-commons/src/external/Share.ts

```diff
@@ -54,16 +55,16 @@ function Share(url?: string, text?: string, title?: string): Promise<boolean> {
                     url
                 },
                 result => resolve(result.completed),
-                error => reject(error)
+                error => reject(new Error(error))
```

Comment:
rename parameter to errorMessage
```suggestion
                errorMessage => reject(new Error(errorMessage))
```

Reply:
Done